scala-interpreter/src/main/scala/org/apache/toree/kernel/interpreter/scala/ScalaInterpreter.scala

```diff
@@ -515,17 +514,12 @@ class ScalaInterpreter() extends Interpreter {
     sparkIMain.beQuietDuring[T](body)
   }
 
-  override def bindSparkContext(sparkContext: SparkContext) = {
+  def bindSparkContext(sparkContext: SparkContext) = {
     val bindName = "sc"
 
     doQuietly {
       logger.debug(s"Binding SparkContext into interpreter as $bindName")
-      bind(
-        bindName,
-        "org.apache.spark.SparkContext",
-        sparkContext,
-        List( """@transient""")
-      )
+      interpret(s"""def ${bindName}: org.apache.spark.SparkContext = kernel.sparkContext""")
```

Comment:
You used `classOf[SQLContext].getName` when binding the SQLContext this way. Do you want to change `org.apache.spark.SparkContext` to `classOf[SparkContext].getName` to be consistent?